### PR TITLE
add sentry support

### PR DESCRIPTION
**Pull request by famartinrh**

### CODE REVIEW

**Review by paoloantinori — APPROVED** [review]

**Comment by EricWittmann** [comment]
Looks good - I'm going to hold off on merging this until we get what's already merged pushed to staging.

**Comment by famartinrh** [comment]
DO NOT MERGE, we may have a compatibility issue

**Pull request merged**
